beforeCarrier.tpl

```diff
@@ -80,12 +80,12 @@
 	<br />
 	<p id="dateofdelivery">
 		{if $nbPackages <= 1}
-			{l s='Approximate date of delivery with this carrier is between' mod='dateofdelivery'}
+			{l s='Approximate date of delivery with this carrier is between' d='Modules.Dateofdelivery.Shop'}
 		{else}
-			{l s='There are %s packages, that will be approximately delivered with the delivery option you choose between' sprintf=$nbPackages mod='dateofdelivery'}
+			{l s='There are %s packages, that will be approximately delivered with the delivery option you choose between' sprintf=$nbPackages d='Modules.Dateofdelivery.Shop'}
 		{/if}
-			<span id="minimal"></span> {l s='and' mod='dateofdelivery'} <span id="maximal"></span> <sup>*</sup>
+			<span id="minimal"></span> {l s='and' d='Modules.Dateofdelivery.Shop'} <span id="maximal"></span> <sup>*</sup>
```

Comment:
Is it possible to do something like 
"Approximate date of delivery with this carrier is between %variable1% and %variable2%"?

"There are %s packages, that will be approximately delivered with the delivery option you choose between %variable1% and %variable2%"

Otherwise I'm afraid it won't be translated correctly ^^

Thanks!